index_creator/core/index_creation.py

```diff
@@ -88,10 +87,16 @@ def generate_json(mydb, file_id = None, release_ver = None):
                 index_doc.dois.add(row['doi'])
             # If this is a new file, then we need to create the initial record and add it to our list of documents
             else:
+                
+                if row['release_version'] == max_release_ver:
+                    row['release_version'] = "Recently Released"
+                else:
+                    row['release_version'] = ""
+                release_version_list.append(row['release_version'])
```

Comment:
Doesn't look like you end up doing anything with this. Can we delete it (and the instantiation)?